django-stubs/http/response.pyi

```diff
@@ -10,6 +10,31 @@ from django.test.client import Client
 from django.urls import ResolverMatch
 from django.utils.datastructures import CaseInsensitiveMapping
 
+_T = TypeVar('_T')
+_U = TypeVar('_U')
+
+class _PropertyDescriptor(Generic[_T, _U]):
```

Comment:
I used https://github.com/python/mypy/issues/3004#issuecomment-493741241 as a reason to test to see if I could implement the property with a descriptor. I'm not sure if there are any negative side effects, but properties *are* descriptors and it looks like the types resolve correctly now (my original code using properties did not work when adding the tests)